### DISCUSSION TITLE
In the Flink engine, how to put multiple statements into the statementset for execution

### DISCUSSION THREAD

**Discussion by waywtdcc** [apache/kyuubi#3193] [Q&A]
In the Flink engine, how to put multiple statements into the statementset for execution? Multiple statements use statementset.addinsertsql (SQL);

**Comment by pan3793** [comment]
cc @link3280 @deadwind4

**Comment by a49a** [comment]
Could you try it on? https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/sqlclient/#execute-a-set-of-sql-statements

**Reply by waywtdcc** [reply]
ok,thank you, is this also supported in flink 1.14.5 and kyuubi 1.5.2 ?
